invenio_formatter/translations/af/LC_MESSAGES/messages.po

```diff
@@ -26,7 +26,7 @@ msgstr ""
 
 #: invenio_formatter/templates/invenio_formatter/macros/badges.html:20
 #: invenio_formatter/templates/semantic-ui/invenio_formatter/macros/badges.html:21
-msgid "reStructedText"
+msgid "reStructuredText"
```

Comment:
please revert all the changes in translation files, the translations are done independently of the development, we only mark the text to be translated
otherwise we might cause issues with the translation workflow